templates/_components/_image.twig

```diff
@@ -3,13 +3,154 @@
 {#
   Image
   
-  Generate image transforms and render images
-  
-  @param   asset        {AssetFileModel}    Image asset to be rendered
-  @param   preset       {string}            Image usage preset
-  @param   class        {string}            Class names to apply to the img tag
-  @param   alt          {string}            Value of the alt attribute for the img tag
-  @param   lazyload     {bool}              Lazyload image or not
+  Generate, transform, manipulate, optimize and render images. 
+
+	What it can do:
+	---------------
+	1. Auto-generates transforms and responsive images via `sizes` and `srcset` attributes 
+	2. Auto-crops images if needed/specified
+	3. Auto-generates art-directed responsive images using the `<picture>` and `<source>` tags
+	4. Auto-generates WebP versions (if supported)
+	5. Lazyload images using lazysizes
+	6. Allows images to be specified as CSS class background images
+	7. Allows use of different source images for different breakpoints
+	8. And finally, renders the image
+
+	How to use:
+  -----------
```

Comment:
Dashes line is not aligned with the first character.

Reply:
Not showing in my editor.
<img width="960" alt="Screenshot 2022-06-23 at 5 43 23 PM" src="https://user-images.githubusercontent.com/30666052/175296036-d9518505-bee9-4c16-befd-8596f1d1419d.png">

Reply:
Maybe because spaces and tabs have been mixed up. We always use spaces in our code, not tabs. Make sure the correct whitespace character is being used. If you cannot see the difference in your editor, try turning on invisible characters.

Reply:
Thank you for the suggestion.